### ISSUE TITLE
翻译的太水

### ISSUE THREAD

**Issue by mozart2019** [jkchao/typescript-book-chinese#110]
How you write TypeScript modules is also a bit of a mess. Again here is how not to do it today:
import foo = require('foo'). i.e. import/require. Use ES module syntax instead.

翻译成了：

怎么书写 TypeScript 模块，这也是一件让人困惑的事。在今天我们应该这么做：
import foo = require('foo') 例如： import/require 使用 ES 模块语法。

各位看看有没有问题

**Comment by justjavac** [comment]
机翻😂

**Comment by jkchao** [comment]
还有其他问题，可以指出来

**Comment by BryanAdamss** [comment]
这一段，我也曾感到困惑😂

**Comment by Mayandev** [comment]
[ThisType](https://jkchao.github.io/typescript-book-chinese/typings/thisType.html) 这一节完全机翻的官方文档。

**Comment by crossjs** [comment]
是的。翻译太水。书买了就丢一边了。最让人反感的是一些所谓的大佬，居然不要脸地在那推荐，还敢把名字印到书上，怀疑到底是不是真的是看过之后才推荐的。

**Comment by syhily** [comment]
买了实体书的也来说一下，翻译的确不容易，但是书里面把基础的反引号都能写错，实在是怀疑是否做到了三审三校和编辑审读。

**Comment by 452392281** [comment]
有些部分的翻译还不如看英文原版，有些机翻的中文词汇、语句看着直接懵逼。
